src/styles/_components/_alerts.scss

```diff
@@ -1,23 +1,35 @@
 @mixin alert() {
+    //note: all the icons svgs are available in the images!
+    @include column(12);
```

Comment:
This might (almost certain it will... xD) break some alerts, as not all of the alerts are used in full-width mode. I would suggest that we remove this and every platform adjusts the width